<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-66CRK13" failures="12" tests="21" name="Test" time="19.068" errors="0" timestamp="2023-05-03T17:41:46 ART">
  <testcase classname="github.SauceDemo.MakeSimpleOrder" name="productSortHiLo" time="1.99"/>
  <testcase classname="github.SauceDemo.MakeSimpleOrder" name="addToCartAndGoToCheckout" time="0.423"/>
  <testcase classname="github.SauceDemo.MakeSimpleOrder" name="checkoutFinish" time="2.194"/>
  <testcase classname="github.SauceDemo.MakeOrderAndDelete" name="productSortZA" time="1.719"/>
  <testcase classname="github.SauceDemo.MakeOrderAndDelete" name="addToCartAndGoToCheckout" time="0.96"/>
  <testcase classname="github.SauceDemo.MakeOrderAndDelete" name="checkoutFinish" time="0.295"/>
  <testcase classname="github.SauceDemo.Login_Chrome" name="login_Firefox_Credentials_Lockout" time="0.366">
    <failure type="java.lang.AssertionError" message="did not expect [https://www.saucedemo.com/] but found [https://www.saucedemo.com/]">
      <![CDATA[java.lang.AssertionError: did not expect [https://www.saucedemo.com/] but found [https://www.saucedemo.com/]
at github.SauceDemo.Login_Chrome.login_Firefox_Credentials_Lockout(Login_Chrome.java:104)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- login_Firefox_Credentials_Lockout -->
  <testcase classname="github.SauceDemo.Login_Chrome" name="login_Firefox_Credentials_None" time="0.199">
    <failure type="java.lang.AssertionError" message="did not expect [https://www.saucedemo.com/] but found [https://www.saucedemo.com/]">
      <![CDATA[java.lang.AssertionError: did not expect [https://www.saucedemo.com/] but found [https://www.saucedemo.com/]
at github.SauceDemo.Login_Chrome.login_Firefox_Credentials_None(Login_Chrome.java:180)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- login_Firefox_Credentials_None -->
  <testcase classname="github.SauceDemo.Login_Chrome" name="login_Firefox_Credentials_Ok" time="0.622"/>
  <testcase classname="github.SauceDemo.Login_Chrome" name="login_Firefox_Credentials_Password_Wrong" time="0.282">
    <failure type="java.lang.AssertionError" message="did not expect [https://www.saucedemo.com/] but found [https://www.saucedemo.com/]">
      <![CDATA[java.lang.AssertionError: did not expect [https://www.saucedemo.com/] but found [https://www.saucedemo.com/]
at github.SauceDemo.Login_Chrome.login_Firefox_Credentials_Password_Wrong(Login_Chrome.java:164)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- login_Firefox_Credentials_Password_Wrong -->
  <testcase classname="github.SauceDemo.Login_Chrome" name="login_Firefox_Credentials_Username_Wrong" time="0.347">
    <failure type="java.lang.AssertionError" message="did not expect [https://www.saucedemo.com/] but found [https://www.saucedemo.com/]">
      <![CDATA[java.lang.AssertionError: did not expect [https://www.saucedemo.com/] but found [https://www.saucedemo.com/]
at github.SauceDemo.Login_Chrome.login_Firefox_Credentials_Username_Wrong(Login_Chrome.java:125)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- login_Firefox_Credentials_Username_Wrong -->
  <testcase classname="github.SauceDemo.Login_Edge" name="login_Firefox_Credentials_Lockout" time="0.379">
    <failure type="java.lang.AssertionError" message="did not expect [https://www.saucedemo.com/] but found [https://www.saucedemo.com/]">
      <![CDATA[java.lang.AssertionError: did not expect [https://www.saucedemo.com/] but found [https://www.saucedemo.com/]
at github.SauceDemo.Login_Edge.login_Firefox_Credentials_Lockout(Login_Edge.java:104)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- login_Firefox_Credentials_Lockout -->
  <testcase classname="github.SauceDemo.Login_Edge" name="login_Firefox_Credentials_None" time="0.189">
    <failure type="java.lang.AssertionError" message="did not expect [https://www.saucedemo.com/] but found [https://www.saucedemo.com/]">
      <![CDATA[java.lang.AssertionError: did not expect [https://www.saucedemo.com/] but found [https://www.saucedemo.com/]
at github.SauceDemo.Login_Edge.login_Firefox_Credentials_None(Login_Edge.java:180)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- login_Firefox_Credentials_None -->
  <testcase classname="github.SauceDemo.Login_Edge" name="login_Firefox_Credentials_Ok" time="0.658"/>
  <testcase classname="github.SauceDemo.Login_Edge" name="login_Firefox_Credentials_Password_Wrong" time="0.302">
    <failure type="java.lang.AssertionError" message="did not expect [https://www.saucedemo.com/] but found [https://www.saucedemo.com/]">
      <![CDATA[java.lang.AssertionError: did not expect [https://www.saucedemo.com/] but found [https://www.saucedemo.com/]
at github.SauceDemo.Login_Edge.login_Firefox_Credentials_Password_Wrong(Login_Edge.java:164)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- login_Firefox_Credentials_Password_Wrong -->
  <testcase classname="github.SauceDemo.Login_Edge" name="login_Firefox_Credentials_Username_Wrong" time="0.38">
    <failure type="java.lang.AssertionError" message="did not expect [https://www.saucedemo.com/] but found [https://www.saucedemo.com/]">
      <![CDATA[java.lang.AssertionError: did not expect [https://www.saucedemo.com/] but found [https://www.saucedemo.com/]
at github.SauceDemo.Login_Edge.login_Firefox_Credentials_Username_Wrong(Login_Edge.java:125)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- login_Firefox_Credentials_Username_Wrong -->
  <testcase classname="github.SauceDemo.Login_Firefox" name="login_Firefox_Credentials_Lockout" time="0.349">
    <failure type="java.lang.AssertionError" message="did not expect [https://www.saucedemo.com/] but found [https://www.saucedemo.com/]">
      <![CDATA[java.lang.AssertionError: did not expect [https://www.saucedemo.com/] but found [https://www.saucedemo.com/]
at github.SauceDemo.Login_Firefox.login_Firefox_Credentials_Lockout(Login_Firefox.java:104)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- login_Firefox_Credentials_Lockout -->
  <testcase classname="github.SauceDemo.Login_Firefox" name="login_Firefox_Credentials_None" time="0.286">
    <failure type="java.lang.AssertionError" message="did not expect [https://www.saucedemo.com/] but found [https://www.saucedemo.com/]">
      <![CDATA[java.lang.AssertionError: did not expect [https://www.saucedemo.com/] but found [https://www.saucedemo.com/]
at github.SauceDemo.Login_Firefox.login_Firefox_Credentials_None(Login_Firefox.java:180)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- login_Firefox_Credentials_None -->
  <testcase classname="github.SauceDemo.Login_Firefox" name="login_Firefox_Credentials_Ok" time="0.446"/>
  <testcase classname="github.SauceDemo.Login_Firefox" name="login_Firefox_Credentials_Password_Wrong" time="0.333">
    <failure type="java.lang.AssertionError" message="did not expect [https://www.saucedemo.com/] but found [https://www.saucedemo.com/]">
      <![CDATA[java.lang.AssertionError: did not expect [https://www.saucedemo.com/] but found [https://www.saucedemo.com/]
at github.SauceDemo.Login_Firefox.login_Firefox_Credentials_Password_Wrong(Login_Firefox.java:164)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- login_Firefox_Credentials_Password_Wrong -->
  <testcase classname="github.SauceDemo.Login_Firefox" name="login_Firefox_Credentials_Username_Wrong" time="0.318">
    <failure type="java.lang.AssertionError" message="did not expect [https://www.saucedemo.com/] but found [https://www.saucedemo.com/]">
      <![CDATA[java.lang.AssertionError: did not expect [https://www.saucedemo.com/] but found [https://www.saucedemo.com/]
at github.SauceDemo.Login_Firefox.login_Firefox_Credentials_Username_Wrong(Login_Firefox.java:125)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- login_Firefox_Credentials_Username_Wrong -->
</testsuite> <!-- Test -->
